### PR TITLE
MMB: Refactor modal stack

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/navigation/NavigationStack.js [hunk 1]
```diff
@@ -78,21 +49,36 @@ const MainStack = StackNavigator(
...
+);
+
 export default StackNavigator(
   {
     MMBMainStack: {
       screen: MainStack,
     },
-    TravelDocumentScreen: {
-      screen: TravelDocumentStack,
+    MyModalStack: {
```
Comment (RobinCsl) [inline app/MMB/src/navigation/NavigationStack.js]: Is the name final?

Comment (tbergq) [reply app/MMB/src/navigation/NavigationStack.js]: Yeah, I was a little bit to fast here. Let me rename that

**Comment by tbergq** [comment]
@RobinCsl 🚀

**Comment by tbergq** [comment]
I know, this is still just preparing to make it good. This is not finished

**Pull request merged**